### ISSUE TITLE
Visually center email-address icon

### ISSUE THREAD

**Issue by macandcheese** [Esri/calcite-ui-icons#146]
🚨 Nitpicky question alert 🚨 

We recently updated the default email input icon in Calcite Components: https://github.com/Esri/calcite-components/pull/865

However the icon itself is visually weighted to the top of its container - so when positioned next to input placeholder / value text, it looks off center.

Is there an adjustment we can make to this icon to reduce that? I like the bottom-right placement (it looks weird if truly centered), but perhaps we could lessen how far the `@` descends (also is there an `@` icon? might be good to have 🤷‍♂️ ):

<img width="402" alt="Screen Shot 2020-08-14 at 10 42 58 AM" src="https://user-images.githubusercontent.com/4733155/90278043-95fb9e80-de1b-11ea-9fd1-f2baa4c8781d.png">

Poor mockup:
<img width="402" alt="Screen Shot 2020-08-14 at 10 42 58 AM" src="https://user-images.githubusercontent.com/4733155/90278465-5b463600-de1c-11ea-9639-e8fdac53d6cd.jpg">

**Comment by KatelynSeitz** [comment]
I agree- your suggestion to move the envelope/@ towards the visual center would be an improvement based on seeing your use case. My guess is this would improve other uses of this icon as well. Thanks for the example!
Below I have mocked up a solution in the middle that could work, where the envelope is moved down a pixel towards the @. This doesn't fill out the artboard as much, but I think aligning with the other envelope icon on right and with the text field is more important here. 

I also adjusted the right to left alignment on the 24 to further center the artwork. The other ones seemed right as is. 
![image](https://user-images.githubusercontent.com/29716057/90280101-31424300-de1f-11ea-9e62-78ba892c5366.png)

Thoughts, @johnmgriffith @macandcheese? Maybe we can include this as a bug fix in our next release?

**Comment by paulcpederson** [comment]
✨ very nice ✨

**Comment by macandcheese** [comment]
Yeah looking good! Maybe bumping the `@` up another px or two would help even more? Either way this is an improvement so I'm happy :)

**Comment by bstifle** [comment]
agreed. 24 could still go up 1px, 32 up 2px. 16 looks perf

**Comment by KatelynSeitz** [comment]
Awesome. Will include this in the next release soon. We can add an @ symbol if that is needed as well. Is there a pattern for using `envelope` https://esri.github.io/calcite-ui-icons/#envelope 
vs. `email-address` https://esri.github.io/calcite-ui-icons/#email-address 
vs. a new `@` icon?

**Comment by johnmgriffith** [comment]
I like these, and I agree with Adam too. Bumping up the `@` a pixel or two so it's more centered might be nice. We can take a look at it. Let's post it as a bug release soon.

**Comment by macandcheese** [comment]
Re: use cases...

We updated from "send" to "email address", which I think is correct. How I see the others:

Envelope: a "new email" or "create email" button,
Send: the final "send email" button,
Email-Address : email input / enter your email address use case.

May not need the "@" icon but I'm sure there's a use case I'm leaving out.

**Comment by johnmgriffith** [comment]
Alright, we are hoping to make a release today, so we'll stick this in there as well. I shifted the `@` symbol up 1 more pixel, so here's a before and after comparison. Black is the old original, and the grey is the new icon:

![image](https://user-images.githubusercontent.com/30184050/90438448-d3b12f00-e088-11ea-8869-edd5d75f31b3.png)

**Comment by johnmgriffith** [comment]
This was just resolved in the latest release!
